doc/nrf/device_guides/working_with_nrf/nrf91/thingy91_gsg.rst

```diff
@@ -63,447 +57,137 @@ Complete the following steps to prepare the Thingy:91 for setup:
+      :alt: Cellular Monitor - Program device
 
-            Programming using External debug probe
+      Cellular Monitor - Program device
 
-      #. Scroll up in the menu on the left to the **FILE** section and click :guilabel:`Clear files`.
+   The **Program sample app** window appears, displaying applications you can program to the Thingy:91.
 
-.. rst-class:: numbered-step
+#. Click :guilabel:`Select` in the **Asset Tracker V2** section.
+   Asset Tracker v2 is an application that simulates sensor data and transmits it to Nordic Semiconductor's cloud solution, `nRF Cloud`_.
 
-.. _update_modem_fw_nrf9160:
+   .. figure:: images/cellularmonitor_selectassettracker.png
+      :alt: Cellular Monitor - Select Asset Tracker V2
 
-Update the modem firmware on the nRF9160 SiP
-============================================
+      Cellular Monitor - Select Asset Tracker V2
 
-.. tabs::
+   The **Program Modem Firmware (Optional)** window appears.
 
-   .. group-tab:: Through USB
+#. Click :guilabel:`Select` in the section for the latest modem firmware.
 
-     To update the modem firmware using USB, complete the following steps:
+   The **Program Mode Firmware (Optional)** window expands to display additional information.
 
-      1. Open `nRF Connect for Desktop`_ and launch the Programmer app if you do not have it open already.
-      #. Make sure that **Enable MCUboot** is selected.
-      #. Switch off the Thingy:91.
-      #. Press **SW3** while switching **SW1** to the **ON** position.
+   .. figure:: images/cellularmonitor_enablemcuboot.png
+      :alt: Cellular Monitor - Enable MCUboot
 
-         .. figure:: images/thingy91_sw1_sw3.svg
-            :alt: Thingy:91 - SW1 SW3 switch
+      Cellular Monitor - Enable MCUboot
 
-            Thingy:91 - SW1 SW3 switch
+#. Switch off the Thingy:91.
+#. Press **SW3** while switching **SW1** to the **ON** position to enable MCUboot mode.
+#. Click :guilabel:`Program` to program the modem firmware to the Thingy:91.
+   Do not unplug or turn off the device during this process.
 
-      #. In the menu, select Thingy:91.
+   When the process is complete, you see a success message.
 
-      #. In the menu on the left, click :guilabel:`Add file` in the **FILE** section, and select :guilabel:`Browse`.
-         A file explorer window appears.
+#. Click :guilabel:`Continue` to move to the next step.
 
-         .. figure:: images/programmer_add_file.png
-            :alt: Programmer - Add file
+   The **Program Mode Firmware (Optional)** changes to the **Program Asset Tracker V2** window.
 
-            Programmer - Add file
+#. Switch off the Thingy:91.
+#. Press **SW3** while switching **SW1** to the **ON** position to enable MCUboot mode.
+#. Click :guilabel:`Program` to program the application to the Thingy:91.
+   Do not unplug or turn off the device during this process.
 
-      .. update_modem_start
+   When the process is complete, you see a success message.
```

Comment:
Should we also mention the failure scenario . _Unable to program the device, please check the log._ 
and tell the user to program again?

Reply:
We don't mention the failure as an option for the nRF9160 DK Getting started, though I guess it might be more likely with the Thingy:91 if MCUboot isn't enabled properly.

Reply:
yeah, I got the message 1-2 times when trying to enable the MCUboot while testing Thingy:91 :)

Reply:
Added a line about switching the Thingy:91 off and enabling MCUboot mode again.